app-insights.tf

```diff
@@ -33,3 +33,11 @@ output "connection_string" {
 output "app_id" {
   value = azurerm_application_insights.this.app_id
 }
+
+output "name" {
+  value = azurerm_application_insights.this.name
+}
+
+output "resource_id" {
```

Comment:
why is this needed?
```suggestion
output "id" {
```

Reply:
needed here - https://github.com/hmcts/sptribs-shared-infrastructure/blob/b5745541432027d9a80e989ee1ad325159084c9e/tf-app-insights-alerts.tf#L23